nnvm/python/nnvm/frontend/keras.py

```diff
@@ -28,6 +28,13 @@ def _get_elu(insym, alpha):
     """
     return -alpha * _sym.relu(1 - _sym.exp(insym)) + _sym.relu(insym)
 
+def _convert_recurrent_activation(insym, keras_layer):
+    if sys.version_info.major < 3:
+        act_type = keras_layer.recurrent_activation.func_name
+    else:
+        act_type = keras_layer.recurrent_activation.__name__
```

Comment:
I think `func.__name__` also works in Python 2.

Reply:
@kazum  Yes.. it has no relation with python2. Thanks.